compose/rest.yaml

```diff
@@ -684,10 +683,14 @@ endpoints:
         type: string
         required: false
         title: Record filtering query
-      - type: map[string]string
+      - type: map[string]any
+        name: meta
+        title: Record meta data
+        parser: payload.ParseMeta
+      - type: map[string]any
         name: labels
-        title: Labels
-        parser: label.ParseStrings
+        title: Labels, deprecated, use meta instead
+        parser: payload.ParseMeta
```

Comment:
Should we just remove it instead of deprecating it? I would vote to just remove it